src/Public/OpenApi.ps1

```diff
@@ -1219,6 +1219,55 @@ function New-PodeOAObjectProperty
     return $param
 }
 
+<#
+.SYNOPSIS
+Creates a OpenAPI schema reference property.
+
+.DESCRIPTION
+Creates a new OpenAPI schema reference from another OpenAPI schema.
+
+.PARAMETER Name
+The Name of the property.
+
+.PARAMETER ComponentSchema
+An component schema name.
+
+.PARAMETER Description
+A Description of the property.
+
+.EXAMPLE
+New-PodeOASchemaProperty -Name 'Config' -ComponentSchema "MyConfigSchema"
+#>
+function New-PodeOASchemaProperty
+{
+    [CmdletBinding()]
+    param(
+        [Parameter()]
+        [string]
+        $Name,
+
+        [Parameter(Mandatory=$true, ParameterSetName = 'ComponentSchema')]
```

Comment:
I added it, because I did not grasp the openapi specs regarding referencing other components like paramters.
It seems not to be allowed, so I'll remove it.